### DISCUSSION TITLE
Error: No Power Available

### DISCUSSION THREAD

**Discussion by mif11k** [dingo35/SmartEVSE-3.5#44] [Q&A]
I've got two SmartEVSEs, one of them behaves strange on 1.8.0 and 3.5.0 firmwares. I cannot figure out what is wrong. The charger refuses to start charging with the error No Power Available. I compared the configuration of both chargers, they are exact the same. Each time when I flash the 1.7.2 the charger starts to deliver power immediately.

The raw output looks like this:

{"version":"v3.5.0-RC2","mode":"SMART","mode_id":3,"car_connected":true,
 "wifi":{"status":"WL_CONNECTED","ssid":"Stenici","rssi":-39,"bssid":"A4:6C:2A:81:B2:B0"},
 "evse":{"temp":32,"temp_max":65,"connected":true,"access":true,"mode":1,"loadbl":0,"pwm":1024,"solar_stop_timer":0,"state":"Charging Stopped","state_id":9,"error":"No Power Available","error_id":1,"rfid":"Not Installed"},
 "settings":{"charge_current":0,"override_current":200,"current_min":7,"current_max":20,"current_main":25,"current_max_circuit":20,"current_max_sum_mains":20,"solar_max_import":0,"solar_start_current":2,"solar_stop_time":33,"enable_C2":"Not present","modem":"Not present","mains_meter":"Sensorbox","starttime":0,"stoptime":0,"repeat":0},
 "mqtt":{"host":"","port":1883,"topic_prefix":"SmartEVSE-11447","username":"","password_set":false,"status":"Disconnected"},"home_battery":{"current":0,"last_update":0},"ev_meter":{"description":"Disabled","address":12,"import_active_power":0,"total_kwh":0,"charged_kwh":0,"currents":{"TOTAL":0,"L1":0,"L2":0,"L3":0},"import_active_energy":0,"export_active_energy":0},"mains_meter":{"import_active_energy":0,"export_active_energy":0},"phase_currents":{"TOTAL":28,"L1":6,"L2":11,"L3":11,"last_data_update":1712248818,"original_data":{"TOTAL":28,"L1":6,"L2":11,"L3":11}},"backlight":{"timer":120,"status":"ON"}}

Once I forced the charger to charge by overriding the charge current, but after I disconnected the car I was unable to resume charging. Any idea what could be wrong?

**Comment by dingo35** [comment]
Please post a telnet log of the erroneous SmartEVSE, from the moment you connect the EV until the error arises.

EDIT: sw version 3.5.0-RC2 please.

**Comment by mif11k** [comment]
Hi, I've got debug output from 1.8.0, I will collect logs from the latest release as soon as the car gets home.

This is from the moment I disconnected the cable and tried to start charging:

[19. 03. 2024, 6:23:49,801 PM] (setState)(C0) 18:23:48 STATE B1 -> A
[19. 03. 2024, 6:23:50,608 PM] (I) (printStatus)(C0) STATE: A Error: 0 StartCurrent: -0 ChargeDelay: 0 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 2.8 A IsetBalanced: 21.2 A
[19. 03. 2024, 6:23:50,974 PM] (I) (printStatus)(C0) L1: 0.8 A L2: 1.4 A L3: 2.8 A Isum: 5.0 A
[19. 03. 2024, 6:23:52,682 PM] (DisconnectEvent)(C0) EV disconnected for a while. Resetting SoC states(printStatus)(C0) STATE: A Error: 0 StartCurrent: -0 ChargeDelay: 0 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 2.8 A IsetBalanced: 21.2 A
[19. 03. 2024, 6:23:52,969 PM] (I) (printStatus)(C0) L1: 0.8 A L2: 1.3 A L3: 2.8 A Isum: 4.9 A
[19. 03. 2024, 6:23:54,425 PM] (ProximityPin)(C0) PP pin: 287 (613 mV)
[19. 03. 2024, 6:23:54,582 PM] (I) (EVSEStates)(C1) Cable limit: 32A  Max: 20A
[19. 03. 2024, 6:23:54,582 PM] (I) (printStatus)(C0) STATE: A Error: 1 StartCurrent: -0 ChargeDelay: 0 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 2.8 A IsetBalanced: 21.2 A
[19. 03. 2024, 6:23:54,978 PM] (I) (printStatus)(C0) L1: 0.8 A L2: 1.3 A L3: 2.8 A Isum: 4.9 A
[19. 03. 2024, 6:23:55,389 PM] (I) (Timer1S)(C0) Not enough current available!
[19. 03. 2024, 6:23:55,980 PM] (setState)(C1) 18:23:55 STATE A -> B1
[19. 03. 2024, 6:23:56,587 PM] (I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -0 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 2.8 A IsetBalanced: 21.2 A
[19. 03. 2024, 6:23:56,986 PM] (I) (printStatus)(C0) L1: 0.7 A L2: 1.1 A L3: 2.8 A Isum: 4.6 A
[19. 03. 2024, 6:23:58,505 PM] (I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -0 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 2.8 A IsetBalanced: 21.2 A
[19. 03. 2024, 6:23:58,993 PM] (I) (printStatus)(C0) L1: 0.8 A L2: 1.3 A L3: 2.8 A Isum: 4.9 A
[19. 03. 2024, 6:24:00,023 PM] (loop)(C1) 0 clients running.
[19. 03. 2024, 6:24:00,566 PM] (I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -0 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 2.8 A IsetBalanced: 21.2 A

**Comment by dingo35** [comment]
Yeah there were hundreds of commits after 1.8.0 so Ill await your RC2 log...

**Reply by mif11k** [reply]
@dingo35 here is the debug output:


Connecting the car:

(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326427.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: A Error: 0 StartCurrent: -2 ChargeDelay: 0 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.4 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.4 A L2: 0.8 A L3: 0.3 A Isum: 2.5 A
(D) (Timer100ms)(C0) ModbusRequest 2: Request MainsMeter Measurement
(D) (ModbusSend8)(C0) Sent packet address: 0x0a, function: 0x04, reg: 0x0000, token:0x0a040000, data: 0x0014.
(D) (ModbusDecode)(C1) Received packet (43 bytes) 0a 04 28 00 10 00 03 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 3f b0 e7 a4 3f 5c 5d 02 3e ba 7b 66
(V) (ModbusDecode)(C1)  valid Modbus packet: Address 0x0a Function 0x04 Register 0x0000 Response
(V) (CalcBalancedCurrent)(C0) Checkpoint 1 Isetbalanced=20.0 A Imeasured=1.3 A MaxCircuit=20 Imeasured_EV=0.0 A, Battery Current = 0.0 A, mode=1.
(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326429.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: A Error: 0 StartCurrent: -2 ChargeDelay: 0 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.3 A Isum: 2.4 A
(ProximityPin)(C0) PP pin: 288 (614 mV)
(I) (EVSEStates)(C1) Cable limit: 32A  Max: 20A
[1;33m(I) (EVSEStates)(C1) Cable limit: 32A  Max: 20A
(I) (Timer1S)(C1) Not enough current available!
(setState)(C1) 16:13:51 STATE A -> B1
(setState)(C1) Pilot Disconnected
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
(D) (Timer100ms)(C0) ModbusRequest 2: Request MainsMeter Measurement
(D) (ModbusSend8)(C0) Sent packet address: 0x0a, function: 0x04, reg: 0x0000, token:0x0a040000, data: 0x0014.
(D) (ModbusDecode)(C1) Received packet (43 bytes) 0a 04 28 00 10 00 03 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 3f b2 e4 00 3f 5d 3f 7f 3e c4 0a 19
(V) (ModbusDecode)(C1)  valid Modbus packet: Address 0x0a Function 0x04 Register 0x0000 Response
(V) (CalcBalancedCurrent)(C0) Checkpoint 1 Isetbalanced=20.0 A Imeasured=1.3 A MaxCircuit=20 Imeasured_EV=0.0 A, Battery Current = 0.0 A, mode=1.
(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326431.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -2 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.3 A Isum: 2.4 A
[0;32m(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326431.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -2 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.3 A Isum: 2.4 A
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
setState)(C1) Switching Contactor2 OFF.
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
(D) (Timer100ms)(C0) ModbusRequest 2: Request MainsMeter Measurement
(D) (ModbusSend8)(C0) Sent packet address: 0x0a, function: 0x04, reg: 0x0000, token:0x0a040000, data: 0x0014.
(D) (ModbusDecode)(C1) Received packet (43 bytes) 0a 04 28 00 10 00 03 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 3f b2 e0 14 3f 62 c1 55 3e cb fd 56
(V) (ModbusDecode)(C1)  valid Modbus packet: Address 0x0a Function 0x04 Register 0x0000 Response
(V) (CalcBalancedCurrent)(C0) Checkpoint 1 Isetbalanced=20.0 A Imeasured=1.3 A MaxCircuit=20 Imeasured_EV=0.0 A, Battery Current = 0.0 A, mode=1.
(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326433.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -2 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.3 A Isum: 2.4 A
[0;32m(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326433.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -2 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.3 A Isum: 2.4 A
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
setState)(C1) Switching Contactor2 OFF.
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
(D) (Timer100ms)(C0) ModbusRequest 2: Request MainsMeter Measurement
(D) (ModbusSend8)(C0) Sent packet address: 0x0a, function: 0x04, reg: 0x0000, token:0x0a040000, data: 0x0014.
(D) (ModbusDecode)(C1) Received packet (43 bytes) 0a 04 28 00 10 00 03 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 3f ae fc 38 3f 66 46 36 3e d6 8a 4d
(V) (ModbusDecode)(C1)  valid Modbus packet: Address 0x0a Function 0x04 Register 0x0000 Response
(V) (CalcBalancedCurrent)(C0) Checkpoint 1 Isetbalanced=20.0 A Imeasured=1.3 A MaxCircuit=20 Imeasured_EV=0.0 A, Battery Current = 0.0 A, mode=1.
(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326435.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -2 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.4 A Isum: 2.5 A
[0;32m(V) (CalcBalancedCurrent)(C0) phaseLastUpdate=1712326435.
(V) (CalcBalancedCurrent)(C0) Checkpoint 2 Isetbalanced=21.4 A, Idifference=5.8, mod=0.
(V) (CalcBalancedCurrent)(C0) Checkpoint 4 Isetbalanced=20.0 A.
(V) (CalcBalancedCurrent)(C0) Checkpoint 5 Isetbalanced=20.0 A.
(I) (printStatus)(C0) STATE: B1 Error: 1 StartCurrent: -2 ChargeDelay: 60 SolarStopTimer: 0 NoCurrent: 0 Imeasured: 1.3 A IsetBalanced: 20.0 A
(I) (printStatus)(C0) L1: 1.3 A L2: 0.8 A L3: 0.4 A Isum: 2.5 A
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
setState)(C1) Switching Contactor2 OFF.
(setState)(C1) Switching Contactor1 OFF.
(setState)(C1) Switching Contactor2 OFF.
(D) (Timer100ms)(C0) ModbusRequest 2: Request MainsMeter Measurement
(D) (ModbusSend8)(C0) Sent packet address: 0x0a, function: 0x04, reg: 0x0000, token:0x0a040000, data: 0x0014.
(D) (ModbusDecode)(C1) Received packet (43 bytes) 0a 04 28 00 10 00 03 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 3f a9 2d 3d 3f 58 39 7d 3e db f8 7d

**Comment by dingo35** [comment]
Your MaxSumMains should be at the default of 600A .

**Comment by mif11k** [comment]
Good, this did the trick! Now I can focus on another issue ;)

M
